puppet/modules/unattended/manifests/init.pp

```diff
@@ -4,14 +4,10 @@
       auto      => { 'reboot' => false },
       blacklist => [
         'docker-ce',
```

Comment:
Unrelated, but we can get rid of this by now. We never really adopted Docker on our CI.

Reply:
Especially, we do not have Debian-based (general purpose) CI nodes ;)